sql_generators/stable_views/__init__.py

```diff
@@ -28,7 +28,10 @@
 AS
 SELECT
   * REPLACE(
-    {replacements})
+    {replacements}),
+  mozfun.norm.extract_version(client_info.app_display_version, 'major') as app_version_major,
```

Comment:
I mentally remembered that these were `STRING`, but it looks like it's `NUMERIC` (per https://github.com/mozilla/bigquery-etl/blob/generated-sql/sql/mozfun/norm/extract_version/udf.sql#L2).  Can you confirm that we can compare with `>=` and friends?

There's a `beta` field as well -- is there any reason to not expose it at the same time?